contracts/strategies/mstable/IdleMStableStrategy.sol

```diff
@@ -291,6 +309,18 @@ contract IdleMStableStrategy is Initializable, OwnableUpgradeable, ERC20Upgradea
         idleCDO = _cdo;
     }
 
+    /// @notice 
+    function _lockedLpTokens() internal view returns (uint256 _locked) {
```

Comment:
Yeah it makes sense, I was thinking also to potentially add this logic in IdleCDO.sol directly, where there is also similar logic for the release of rewards sold by IdleCDO directly during harvests so to avoid logic duplication in all strategies, but I guess it would complicate the logic too much. wdyt?